firmware/lucidgloves-firmware/AdvancedConfig.h

```diff
@@ -4,8 +4,13 @@
 #define CALIBRATION_LOOPS -1//How many loops should be calibrated. Set to -1 to always be calibrated.
 
 //Comm defines, no touchy
-#define COMM_SERIAL 0   
-#define COMM_BTSERIAL 1 
+#define COMM_SERIAL 0
+#define COMM_BTSERIAL 1
+#define COMM_BLE 2
+
+#define BLE_SERVICE_UUID "090b69e6-d509-4c32-8b20-5f8b947b253a"
```

Comment:
I recommend using standard chars for uart: https://infocenter.nordicsemi.com/index.jsp?topic=%2Fcom.nordic.infocenter.sdk5.v13.0.0%2Fble_sdk_app_nus_eval.html
It is widely supported, you can download NRF connect for your phone, I use it quite a lot in my project